scst-store/amr/auth-k8s-sa-autoconfiguration.md

```diff
@@ -1,24 +1,23 @@
 # Kubernetes service account autoconfiguration
 
-This site is giving an overview about the resources that get created for the convenience feature of kubernetes service account autoconfiguration for AMR authentication and authorization. It will help you nderstand which resource play a role in the default configuration and where to look at in case issues arise. If you need a more deep understanding of the resources and if you are in need to set up these resources on your own, for example if you use means to manage your service accounts or have other requirements related to roles and bindings head over to [User-defined kubernetes service account configuration](scst-store/amr/auth-k8s-sa-user-defined.hbs.md)
+This site gives an overview of the resources created for the convenience feature of kubernetes service account autoconfiguration for AMR authentication and authorization. It will help you understand which resource plays a role in the default configuration and where to look in case issues arise. If you need a deeper understanding of the resources and if you are in need to set up these resources on your own, for example, if you use means to manage your service accounts or have other requirements related to roles and bindings head over to [User-defined kubernetes service account configuration](scst-store/amr/auth-k8s-sa-user-defined.hbs.md)
 
-All of the package-level configuration is prefixed with a component top level key (TLK) and can also be inflenced by certain TAP level configuration, which will be mentioned in the seperate sections.
+All package-level configurations are prefixed with a component top level key (TLK) and can also be influenced by certain TAP level configurations, which will be mentioned in the separate sections.
 
 
 ## Observer
 
 Observer configuration in the TAP context is prefixed with the TLK `amr.observer`. 
-For authentication and authorization the TAP profiles will influence autconfiguration. Observer can only autoconfigure itself when colocated with the cloudevent handler, which is currently defined as being in `full` or `view` profile. If this is not the case `auth.kubernetes_service_accounts.autoconfigure` will be set to false at install time.
+For authentication and authorization, the TAP profiles will influence autoconfiguration. The observer can only autoconfigure itself when colocated with the cloudevent handler, which is currently defined as being in `full` or `view` profile. If this is not the case `auth.kubernetes_service_accounts.autoconfigure` will be set to false at install time.
```

Comment:
`auth.kubernetes_service_accounts.autoconfigure` would need to be set to false for any profile that is not full or view, right? It doesn't happen automatically does it? If it is not automatic, can you rephrase the last sentence to something like:

```
If it is not being installed in the `full` or `view` profile, `auth.kubernetes_service_accounts.autoconfigure` should be set to false in the `values.yaml` file. 
```

Reply:
when installed as part of TAP, the value is automatically set to false in the PackageInstall resources for amr-observer. So this should be correct as is.